libs/admin/src/lib/components/users/users.component.scss

```diff
@@ -0,0 +1,32 @@
+@use "../../styles/variables";
+
+::ng-deep .workgroups-tooltip {
+  font-size: 12px;
+}
+
+.users {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  padding: 1rem;
+}
+
+.users__header {
+  display: flex;
+  justify-content: start;
+  align-items: center;
+  margin-bottom: 1rem;
+}
+
+.users__table {
```

Comment:
Remove the `users__` prefix on all classes in this component as every element in this component's template is wrapped inside the `.users` div anyways.